.pre-commit-config.yaml

```diff
@@ -0,0 +1,38 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: check-merge-conflict
+      - id: check-symlinks
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+
+  - repo: https://github.com/pycqa/isort
+    rev: 5.13.2
+    hooks:
+      - id: isort
+        # Tell isort to output in a Black-compatible style, otherwise you
+        # can get into a situation where isort changes a file and Black changes
+        # it back, so pre-commit never passes.
```

Comment:
I think use of pronouns (such as  "you" here) is not best practice to write comments IMO and it also reflects sometime informal and directive. We can change the above comment something to like _(Just a suggestion)_:

```
# Configure isort to use Black's import style to prevent formatting conflicts.
# As isort and Black may repeatedly reformat the same code, causing pre-commit to fail.
```

Reply:
Done